gamestonk_terminal/loggers.py

```diff
@@ -0,0 +1,181 @@
+"""Terminal helper"""
+__docformat__ = "numpy"
+import logging
+import os
+from pathlib import Path
+import sys
+import time
+import uuid
+
+import git
+
+import gamestonk_terminal.config_terminal as cfg
+
+logger = logging.getLogger(__name__)
+LOGFORMAT = "%(asctime)s|%(levelname)s|%(name)s|%(funcName)s|%(lineno)s|%(message)s"
+DATEFORMAT = "%Y-%m-%dT%H:%M:%S%z"
+
+
+def library_loggers(verbosity: int = 0) -> None:
+    """Setup library logging
+
+    Parameters
+    ----------
+    verbosity : int, optional
+        Log level verbosity, by default 0
+    """
+
+    logging.getLogger("requests").setLevel(
+        logging.INFO if verbosity <= 1 else logging.DEBUG
+    )
+    logging.getLogger("urllib3").setLevel(
+        logging.INFO if verbosity <= 1 else logging.DEBUG
+    )
+    logging.getLogger("ccxt.base.exchange").setLevel(
+        logging.INFO if verbosity <= 2 else logging.DEBUG
+    )
+
+
+def setup_file_logger() -> None:
+    """Setup File Logger"""
+
+    file_path = Path(__file__)
+    logger.debug("Parent dir: %s", file_path.parent.parent.absolute())
+    log_dir = file_path.parent.parent.absolute().joinpath("logs")
+    logger.debug("Future logdir: %s", log_dir)
+
+    if not os.path.isdir(log_dir.absolute()):
+        logger.debug("Logdir does not exist")
+        os.mkdir(log_dir.absolute())
+
+    log_id = log_dir.absolute().joinpath(".logid")
+
+    if not os.path.isfile(log_id.absolute()):
+        logger.debug("Log ID does not exist: %s", log_id.absolute())
```

Comment:
Can we use f-string convention here?

Reply:
Pylint explicitly advises against using f-strings with logging
https://pycodequ.al/docs/pylint-messages/w1203-logging-fstring-interpolation.html

Reply:
Wow